.github/workflows/flutter.yml

```diff
@@ -59,15 +63,21 @@ jobs:
           env:
             SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
           run: |
+            flutter config --enable-windows-desktop
+            flutter config --enable-macos-desktop
+            flutter config --enable-linux-desktop
             cd flutter/example
             TARGET=${{ matrix.target }}
             flutter pub get
             case $TARGET in
             ios)
-            flutter build ios --release --no-codesign
+            flutter build ios --no-codesign
+            ;;
+            macos)
+            flutter build macos
```

Comment:
maybe this requires `--no-codesign` too?

Reply:
> flutter build macos --no-codesign
> Could not find an option named "codesign".

Surprisingly no